client/me/concierge/shared/no-available-times.js

```diff
@@ -26,24 +27,29 @@ class NoAvailableTimes extends Component {
 				<PrimaryHeader isWhiteGlove={ isWhiteGlove } />
 				<Card>
 					<h2 className="shared__no-available-times-heading">
-						{ translate( 'Sorry, there are no sessions available' ) }
+						{ translate( 'Sorry, all upcoming sessions are full.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Sorry, all upcoming sessions are full' )` *ES Score: 12*

Reply:
@donlair Just noting that the period makes it a new translation. It's okay if that's what you wanted. If you want to get the translations without waiting for the vendor, you could just head to translate.wordpress.com, search for this string, and the period at the end of each translation.